.gitignore

```diff
@@ -1,2 +1,5 @@
 /build
 /node_modules
+.idea
+data/srtm-1arcsec-ro.tiff
```

Comment:
Just ignore the whole `/data` folder (leading slash, so it only matches the `data` folder in the project root). And make sure there is a newline at the end of the file.